VocaDbWeb.Core/Startup.cs

```diff
@@ -50,7 +57,29 @@ public Startup(IConfiguration configuration)
 		// This method gets called by the runtime. Use this method to add services to the container.
 		public void ConfigureServices(IServiceCollection services)
 		{
-			services.AddControllersWithViews();
+			services
+				.AddControllersWithViews(options =>
+				{
+					// Code from: https://stackoverflow.com/questions/51411693/null-response-returns-a-204/60858295#60858295
+					// remove formatter that turns nulls into 204 - No Content responses
+					// this formatter breaks Angular's Http response JSON parsing
+					options.OutputFormatters.RemoveType<HttpNoContentOutputFormatter>();
+
+					options.Filters.Add<VoidAndTaskTo204NoContentFilter>();
+				})
+				.ConfigureApiBehaviorOptions(options =>
+				{
+					// Code from: https://docs.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-5.0#apicontroller-attribute
+					options.SuppressModelStateInvalidFilter = true;
+				})

```

Comment:
Disabled automatic 400 response.

Reply:
You should mention in code comments why is this here.

Reply:
By the way, why is it sending CategoryName?

Reply:
Because [TagAutoComplete.ts](https://github.com/VocaDB/vocadb/blob/e96b44e6c7970e5300e6780aad2d39cf91a65509/VocaDbWeb/Scripts/KnockoutExtensions/TagAutoComplete.ts#L26) is sending it. We should remove it at some point though.

Reply:
Sounds like it would be very easy to fix TagAutoComplete not to send it.

Reply:
I've removed `CategoryName`. Should we re-enable automatic 400 response?

Reply:
I think it would be best to re-enable it for now. Going with default config is always best if possible. And it may help us notice other issues, that's the intention anyway.